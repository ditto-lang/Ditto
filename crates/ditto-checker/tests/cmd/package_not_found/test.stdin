module Test exports (..)

import (wut) Huh (nah)


five : Float = 5.0
