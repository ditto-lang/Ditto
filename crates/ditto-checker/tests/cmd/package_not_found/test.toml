bin.name = "ditto-checker-testbin-check-module"
args = ["package_not_found.ditto"]
fs.sandbox = true
status = "failed"
