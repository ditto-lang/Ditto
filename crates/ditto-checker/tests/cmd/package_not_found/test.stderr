
  × package not found
   ╭─[package_not_found.ditto:2:1]
 2 │ 
 3 │ import (wut) Huh (nah);
   ·         ─┬─
   ·          ╰── not installed
 4 │ 
   ╰────
  help: try adding `wut` to your dependencies?

